pkg/schedule/schedulers/hot_region_rank_v1.go

```diff
@@ -29,9 +29,10 @@ func initRankV1(r *balanceSolver) *rankV1 {
 }
 
 // isAvailable returns the solution is available.
-// The solution should have no revertRegion and progressiveRank < 0.
+// The solution should progressiveRank > 0.
```

Comment:
Does the comment in `type solution struct` about `progressiveRank` also need to be changed?

Reply:
good catch